BigQuery/tests/System/BigQueryTestCase.php

```diff
@@ -52,15 +53,29 @@ public static function setUpBeforeClass(): void
             'keyFilePath' => $keyFilePath
         ]);
         self::$dataset = self::createDataset(self::$client, uniqid(self::TESTING_PREFIX));
-        self::$table = self::createTable(self::$dataset, uniqid(self::TESTING_PREFIX));
+        $options['schemaFile'] = '/data/table-schema.json';
+        self::$table = self::createTable(self::$dataset, uniqid(self::TESTING_PREFIX), $options);
+        $options['schemaFile'] = '/data/legacy-table-schema.json';
+        self::$legacyTable = self::createTable(self::$dataset, uniqid(self::TESTING_PREFIX), $options);
```

Comment:
What's the need for a legacytable when there's the same table with an extra JSON field?

Reply:
Can we modify the tests to make it work with just one table?

Reply:
Actually the legacy query does not work with table with Json column data type. This is the error:
```
"message": "Querying tables with INTERVAL, JSON, or RANGE type is not supported in Legacy SQL"
```